pkg/bufferdecoder/protocol.go

```diff
@@ -20,6 +20,8 @@ const (
 // keep the 1-byte 'Argnum' as the final parameter before the padding (if padding is needed).
 type Context struct {
 	Ts          uint64
+	ThreadTs    uint64
+	ProcTs      uint64
```

Comment:
Why do we need this data as part of the context of each event?
This is only required for new processes, isn't it?
If that so, adding it to sched_process_fork/sched_process_exec should have been enough, isn't that so?